api/src/main/java/org/apache/cloudstack/api/command/user/firewall/CreatePortForwardingRuleCmd.java

```diff
@@ -294,8 +299,8 @@ public long getNetworkId() {
             ntwkId = networkId;
         }
         if (ntwkId == null) {
-            throw new InvalidParameterValueException("Unable to create port forwarding rule for the ipAddress id=" + ipAddressId +
-                    " as ip is not associated with any network and no networkId is passed in");
+            throw new InvalidParameterValueException("Unable to create port forwarding rule for ip address '" + getIpAddressUuid() +
```

Comment:
I would say: "for the IP Address with ID = '" + getIpAddressUuid() + "'...."

Because just saying "iip address '" + getIpAddressUuid() + "' doesn't make clear that is an ID and not the IP itself. Could be misleading.